toolkit/tools/pkg/simplechroottool/simplechroottool.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// A common interface for creating a basic spec/rpm parsing environment inside a chroot.
+
+package simplechroottool
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/rpm"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
+)
+
+const (
+	chrootSpecDirPath = "/SPECS"
+)
+
+// SimpleChrootTool is a tool for creating an environment inside a chroot suitable for basic tasks like parsing RPMs.
+// The following fields will be set by InitializeChroot():
+//   - chroot: The chroot environment we are working in, call SimpleChrootTool.RunInChroot() to execute commands inside the chroot.
+//   - chrootSpecDir: The relative path to the SPECS directory inside the chroot: "/SPECS."
+//   - runChecks: Whether or not to parse with check sections enabled.
+//   - distTag: The distribution tag to use when parsing RPMs.
+type SimpleChrootTool struct {
+	chroot        *safechroot.Chroot
+	chrootSpecDir string
+	runChecks     bool
+	distTag       string
+}
+
+// ChrootRootDir returns the root directory where the chroot was created. Call InitializeChroot() before calling this function.
+func (s *SimpleChrootTool) ChrootRootDir() string {
+	if s.chroot == nil {
+		return ""
+	}
+	return s.chroot.RootDir()
+}
+
+// ChrootSpecDir returns the directory inside the chroot where the specs dir is mounted. Call InitializeChroot() before calling this function.
+func (s *SimpleChrootTool) ChrootRelativeSpecDir() string {
```

Comment:
>ChrootRelativeSpecDir [](http://example.com/codeflow?start=27&length=21)

Nit-pick: I guess `ChrootSpecDir` would already be clear enough. The word "relative" might suggest that the path is a relative one within the context of the chroot.

Reply:
Open to naming suggestions. I want to be clear that this is the '/SPECS/' part of '/path/to/chroot/**SPECS**/' and not the rest of the root folder.